scripts/run_backend_tests.py

```diff
@@ -241,8 +251,11 @@ def main(args=None):
         # https://stackoverflow.com/q/10095037 for more details.
         sys.path.insert(1, directory)
 
-    import dev_appserver
-    dev_appserver.fix_sys_path()
+    if 'google' in sys.modules:
```

Comment:
When won't this be true? Also, I suggest adding comments explaining why this is necessary.